ClimbingRoutes.js

```diff
@@ -11,18 +11,35 @@ import {
 } from 'native-base';
 
 class climbingRoutes extends Component {
+  state = {
+    routeList: [],
+  };
+
+  componentDidMount() {
+    axios
+      .get('https://climbar.herokuapp.com/api/climbingroutes')
+      .then(res => res.data)
+      .then(data => {
+        console.log(data);
+        this.setState({routeList: data});
```

Comment:
I guess you're gonna switch this to redux? (Sorry if this is an old commit.)

Reply:
Yep, working on it.